pkg/virt-operator/resource/apply/apps.go

```diff
@@ -54,6 +54,8 @@ func (r *Reconciler) syncDeployment(origDeployment *appsv1.Deployment) (*appsv1.
 		replicas := int32(*kv.Spec.Infra.Replicas)
 		if deployment.Spec.Replicas == nil || *deployment.Spec.Replicas != replicas {
 			deployment.Spec.Replicas = &replicas
+			log.Log.Warningf("applying custom number of infra replica. this is an advanced feature that prevents " +
```

Comment:
I am not sure how many users will be looking into logs. Maybe you want to use event?